sysutils/apcupsd/src/opnsense/service/conf/actions.d/actions_apcupsd.conf

```diff
@@ -0,0 +1,29 @@
+[start]
+command:/usr/sbin/service apcupsd start && sleep 2
+parameters:
+type:script_output
```

Comment:
```suggestion
type:script
```

Reply:
Think this should be ok.  Don't think we need to catch the output on this.  Status could always be checked with "status" command later.
//Dan Lundqvist (mrzaz)

Reply:
Are you using it? From the mutable controllers none of this is ever read on start/stop/restart.

Reply:
To be honest, I think this was included in the initial script before I came aboard so have not thought about it.
But doubt that we need the output on this.
What do you say [xbb](https://github.com/xbb) ?
//Dan Lundqvist (mrzaz)

Reply:
I don't think `script_output` is needed except for the status actions